packages/editor/src/client/blocks/richTextBlock/richTextBlock.tsx

```diff
@@ -134,6 +133,12 @@ export const RichTextBlock = memo(function RichTextBlock({
 
             <ToolbarDivider />
 
+            <EditorSubMenuButton icon="font" editorHasFocus={hasFocus}>
+              <FormatColor />
```

Comment:
Can you set the whole modal inside the submenu instead of the button?

Reply:
@penk85 I made some changes, which should let the color picker menu stay open even after clicking on the palette. As well changing the Icon and used references.